example/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
```

Comment:
Like @JakeWharton mentioned, the `android-apt` plugin is only for android projects, but you can use the standard apt plugin for java projects.